test/helpers.js

```diff
@@ -60,8 +60,8 @@ exports.formatFns = {
     return info;
   },
 
-  assign(info) {
-    return Object.assign({}, info);
+  assign(info, opts) {
```

Comment:
not strictly necessary but 
As I needed, for test purpose, a method that returned a new object with additional keys, instead of creating a new helper, I preferred to increase the `assign` functionalities making it, at the same time, more aligned to its 'name'.
As it was before, in fact, more than an `assign` that would have been a `shallowClone` method.